config/stack/manifests/install.yaml

```diff
@@ -1,23 +1,23 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: "crossplaneio-minimal-gcp"
+  name: "crossplaneio-stack-minimal-gcp"
   labels:
-    core.crossplane.io/name: "crossplaneio-minimal-gcp"
+    core.crossplane.io/name: "crossplaneio-stack-minimal-gcp"
 spec:
   selector:
     matchLabels:
-      core.crossplane.io/name: "crossplaneio-minimal-gcp"
+      core.crossplane.io/name: "crossplaneio-stack-minimal-gcp"
   replicas: 1
   template:
     metadata:
-      name: "crossplaneio-minimal-gcp-controller"
+      name: "crossplaneio-stack-minimal-gcp-controller"
       labels:
-        core.crossplane.io/name: "crossplaneio-minimal-gcp"
+        core.crossplane.io/name: "crossplaneio-stack-minimal-gcp"
     spec:
       containers:
-      - name: "crossplaneio-minimal-gcp-controller"
-        image: "crossplaneio/minimal-gcp:1.0"
+      - name: "crossplaneio-stack-minimal-gcp-controller"
+        image: "crossplaneio/stack-minimal-gcp:latest"
```

Comment:
we'll need to think through more how we version and publish these minimal stacks.  The stacks that use the build submodule have a nice versioning story figured out but for the stacks that do not use the build submodule we'll need to figure it out still.  I don't think there's proper automation in place to have the version number injected into the build when the CI system runs it.